lib/index.js

```diff
@@ -36,20 +41,88 @@ class AttributeList {
 
     this.el.setAttribute(this.attributeName, this.tokenList.join(" "))
   }
-}
 
-export function $al(el) {
-  function toCamelCase(str){
-    return str.replace(/^([A-Z])|[\s-_]+(\w)/g, (match, p1, p2, offset) =>  p2 ? p2.toUpperCase() : p1.toLowerCase());
+  remove(attr){
+    var parsed = AttributeList.parseAttributeFakeTokenList(arguments, 'add');
+    parsed.forEach(token => {
+      let pos = this.tokenList.indexOf(token)
+      if (pos > -1){
+        this.tokenList.splice(pos, 1)
+      }
+    })
+
+    this.el.setAttribute(this.attributeName, this.tokenList.join(" "))
+  }
+
+  toggle(attr){
+    let pos = this.tokenList.indexOf(attr)
+    if (pos > -1){
+      this.remove(attr)
+    } else {
+      this.add(attr)
+    }
+    this.el.setAttribute(this.attributeName, this.tokenList.join(" "))
   }
+}
+
+
+// Helper
+function toCamelCase(str){
+  return str.replace(/^([A-Z])|[\s-_]+(\w)/g, (match, p1, p2, offset) =>  p2 ? p2.toUpperCase() : p1.toLowerCase());
+}
+
+
+/*
+ * @name $attributeList
+ * @description AttributeList Wrapper Mode, se like _loadash or jquery
+ *
+ * @param {HTMLElement|String} el — target element
+ */
+export function $attributeList(el) {
```

Comment:
i'm creating a new instance everytime i call this function. I know that i reset `alObj ={}` but maybe theres a better way of doing this.

1. Run the function once, and extending the prototype for that element, that way we could store the instance there. Cons if new attributes were added we couldn't manipulate them.